exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,7 @@
 // Write your function here
-
+function createGreeting(name){
+    return ("Nice to meet you " + name)
```

Comment:
Here you've wrapped the return value in parentheses.
Do you need these parentheses in order for this function to work?